<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bodoni MT</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar
{
	background-color:#F0F1EF; 
	width: 10px;
	margin: 10px 0 10px 0;
}
QScrollBar::handle:vertical
{
	background-color:rgba(214,214,214,70%);
    width:9px;
	min-height: 20px;
    border-radius:5px;
}
QScrollBar::handle:vertical:hover
{
	background-color:rgba(200,200,200,95%);
    width:9px;
	min-height: 20px;
    border-radius:5px;
}
QScrollBar::handle:vertical:pressed
{ 
	background-color:rgba(180,180,180,100%);
    width:9px;
	min-height: 20px;
    border-radius:5px;
}
QScrollBar::add-line:vertical 
{
background:rgb(247,247,247);
border-image: url(:/icon/down.png); 
height: 10px; 
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical 
{
background:rgb(247,247,247);
border-image: url(:/icon/up.png);
height: 10px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar::add-page:vertical
{
  	background-color:rgb(247,247,247);
}
QScrollBar::sub-page:vertical 
{
    background-color:rgb(247,247,247);
}
QScrollBar:horizontal 
{
      border: 0px solid grey;
      background: #F0F1EF;
      height: 10px;
      margin: 0px 10px 0 10px;
}
QScrollBar::handle:horizontal {
     background: rgb(214,214,214,70%);
      min-width: 20px;
	border-radius:5px;
}
QScrollBar::handle:horizontal:hover
{
	background-color:rgba(200,200,200,95%);
    width:9px;
	min-height: 20px;
   border-radius:5px;
}
QScrollBar::handle:horizontal:pressed
{ 
	background-color:rgba(180,180,180,100%);
    width:9px;
	min-height: 20px;
    border-radius:5px;
}
  QScrollBar::add-line:horizontal {
      border-image: url(:/icon/right.png); 
      background: rgb(247,247,247);
      width: 10px;
      subcontrol-position: right;
      subcontrol-origin: margin;
  }
QScrollBar::sub-line:horizontal 
{
      border-image: url(:/icon/left.png); 
      background: rgb(247,247,247);
      width: 10px;
      subcontrol-position: left;
      subcontrol-origin: margin;
 }
</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="View">
    <property name="title">
     <string>视图(&amp;M)</string>
    </property>
    <addaction name="actionclear"/>
   </widget>
   <widget class="QMenu" name="Tool">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="actionEditHardware"/>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_description"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="File"/>
   <addaction name="View"/>
   <addaction name="Tool"/>
   <addaction name="Help"/>
  </widget>
  <widget class="MToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>ToolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="MStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <action name="actionOpenFile">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="actionEditHardware">
   <property name="text">
    <string>板编辑器</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>调试窗口</string>
   </property>
  </action>
  <action name="action_description">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MToolBar</class>
   <extends>QToolBar</extends>
   <header location="global">mclass/mtoolbar.h</header>
  </customwidget>
  <customwidget>
   <class>MStatusBar</class>
   <extends>QStatusBar</extends>
   <header location="global">mclass/mstatusbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
